model/historical_test_data.go

```diff
@@ -61,130 +72,150 @@ func (d *HistoricalTestData) Setup(e cedar.Environment) { d.env = e }
 // IsNil returns if the HistoricalTestData is populated or not.
 func (d *HistoricalTestData) IsNil() bool { return !d.populated }
 
-// Find searches the globally configured pail bucket for the
-// HistoricalTestData. The enviromemt should not be nil.
+// Find searches the database for the HistoricalTestData. The enviromemt should
+// not be nil.
 func (d *HistoricalTestData) Find(ctx context.Context) error {
```

Comment:
Can we specify in the doc comment that this only finds by ID?

Reply:
Done.